### PR TITLE
cp-163: Add hover effect over buttons on the Survey page

### PR DESCRIPTION
**Pull request by OlenaYatseniuk**
- Added hover styling to checkbox and button on the Survey page

image
image

### CODE REVIEW

**Comment by liza-veis** [comment]
can we just add a background color similar to selected items without borders?

**Comment by OlenaYatseniuk** [comment]
> can we just add a background color similar to selected items without borders?

Yes, if you want, I can change it. For button and checkbox or only checkbox?

**Comment by OlenaYatseniuk** [comment]
image


> can we just add a background color similar to selected items without borders?
Done. In the checkbox, the first item - is checked so I left a blue color to have differences with the last which only is hovered. Do you like it?
image
image

**Comment by liza-veis** [comment]
Yes, looks good

File: frontend/src/assets/css/variables.scss [hunk 1]
```diff
@@ -18,4 +19,6 @@
   --blue-alpha-70: #a6d4ffb2;
   --blue-alpha-30: #527deb4d;
   --gray-alpha-50: #302e3680;
+  --secondary-button-shadow: 0px 2px 4px -2px #002c840f,
```
Comment (liza-veis) [inline frontend/src/assets/css/variables.scss]: this is too specific, define it where it's used

File: frontend/src/libs/components/checkbox/styles.module.scss [hunk 2]
```diff
@@ -11,6 +11,11 @@
   cursor: pointer;
 }
 
+.container:hover {
+  // border: 1px solid var(--blue-700)
```
Comment (liza-veis) [inline frontend/src/libs/components/checkbox/styles.module.scss]:
```suggestion
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Review by fshabanov — APPROVED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**